src/PdfReader/index.tsx

```diff
@@ -248,6 +264,50 @@ export default function usePdfReader(args: ReaderArguments): ReaderReturn {
     }
   }, [containerSize, state.pdfWidth, state.pdfHeight, resizePage]);
 
+  /**
+   * Hide Or Show Page Button
+   */
+  React.useEffect(() => {
+    if (!manifest) return;
+
+    const isSinglePDF = manifest?.readingOrder.length <= 1;
```

Comment:
I would extract this const out to the top level of the hook, as it seems we will probably be needing it in other places